frontend/public/components/row-filter.jsx

```diff
@@ -164,5 +164,5 @@ class CheckBoxes_ extends React.Component {
   }
 }
 
-/** @type {React.SFC<{items: Array, itemCount: number, numbers: any, reduxIDs: Array, selected?: Array, type: string, applyFilter?: (filter) => void}>} */
+/** @type {React.SFC<{items: Array, itemCount: number, numbers: any, reduxIDs: Array, selected?: Array, type: string, applyCustomFilter?: (filter) => void}>} */
```

Comment:
Ideally we've give a type to the `filter` arg. Is it a `string[]`?

Reply:
Added type here.